scripts/setup.py

```diff
@@ -215,7 +209,7 @@ def run_custom_scripts():
         for script in custom_scripts:
             if "/controller.d/" in str(script):
                 timeout = lkp.cfg.get("controller_startup_scripts_timeout", 300)
-            elif "/compute.d/" in str(script):
+            elif "/compute.d/" in str(script) or "/nodeset.d/" in str(script):
```

Comment:
It looks like it will run both main compute script and nodeset script if both are defined. Order is dependent on order of custom_scripts variable.

Is there any way to get a deterministic order. I would think we would want main compute script to run first.

Reply:
> Order is dependent on order of custom_scripts variable.

> Is there any way to get a deterministic order.   I would think we would want main compute script to run first.

```py
custom_dirs = [custom_dir / "compute.d", custom_dir / "nodeset.d"]
```

Already done.